home-manager/home.nix

```diff
@@ -143,6 +143,13 @@
     irb-power_assert
   '';
 
+  # typos does not have feature global config, this is used in git hooks for https://github.com/kachick/dotfiles/issues/412
+  xdg.configFile."typos/_typos.toml".text = ''
```

Comment:
This may be clarified when renamed with the each purpose... But okay for now(I'm tired)